yellow-paper/docs/addresses-and-keys/specification.md

```diff
@@ -824,31 +631,11 @@ $\ovskapp$ | See derivations above. Use the decrypted app_address. | |
 $\happenc$ | h("?", $\ovskm$, $\Epkd$) |
 $\plaintext$ | $decrypt_{\happenc}(\ciphertext)$ |
 
-> TODO: how does a user validate that they have successfully decrypted a ciphertext? Is this baked into ChaChaPoly1035, for example?
+<!-- TODO: how does a user validate that they have successfully decrypted a ciphertext? Is this baked into ChaChaPoly1035, for example? -->
 
-### Doing this inside an app circuit
-
-Here's how an app circuit could constrain the app-siloed outgoing viewing secret key ($\ovskapp$) to be correct:
-
-The app circuit exposes, as public inputs, an "outgoing viewing key validation request":
-
-<!-- prettier-ignore -->
-| Thing | Derivation | Name | Comments |
-|---|---|---|---|
-`outgoing_viewing_key_validation_request` | app_address: app_address,<br />hardened_child_sk: $\nskapp$,<br />claimed_parent_pk: $\Npkm$ |
-
-The kernel circuit can then validate the request (having been given $\ovskm$ as a private input to the kernel circuit):
-
-<!-- prettier-ignore -->
-| Thing | Derivation | Name | Comments |
-|---|---|---|---|
-$\ovskapp$ | $h(\ovskm, \text{app\_address})$ |
-$\Ovpkm$ | $\ovskm \cdot G$ | Outgoing viewing public key |
-| | | | Copy-constrain $\ovskm$ with $\ovskm$. |
-
-If the kernel circuit succeeds in these calculations, then the $\ovskapp$ has been validated as the correct app-siled secret key for $\Ovpkm$.
```

Comment:
Commenting here even though it just got merged (sorry!)
This deleted section is still relevant, right?

Reply:
Are we going with this approach, or with the `pokodl_request` described at the end?

Reply:
I think probably this approach (rather than the pokodl approach). It's more conventional and more closely aligned with bip32.

Reply:
Added back this approach and removed the appendix with the alternative key derivation and pokodl request in https://github.com/AztecProtocol/aztec-packages/pull/3837.